packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -794,7 +794,11 @@ async function handler(
   const seed = `${organizerUser.username}:${dayjs(reqBody.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
-  const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
+  let bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
```

Comment:
getLocationValueForDB returns locationBodyString if no location is found in the event type